### DISCUSSION TITLE
超时错误/Timeout Error

### DISCUSSION THREAD

**Discussion by Sydney2580** [kitsuned/PotatoNV#143] [Q&A]
PotatoNV v2.2.1
User manual: https://kutt.it/pnv-en
Verifying images...
Uploading hisi65x_a...
- xloader
- fastboot
Waiting for any device...
Timeout error.

**Comment by bzy-080408** [comment]
你确定手机已经进入com1.0模式了吗？ 看一看设备管理器

**Reply by Sydney2580** [reply]
之前我记得是进入了的，后面的解决方法是全部卸载重新装驱动全部重新来过，反正这个驱动很迷，总而言之后面重新推翻来过应该是驱动也打好了，其他的也弄好了，就成功了，我发这个评论的4月前那次是怎么试都成功不了，并且也是进入了com1.0模式

**Comment by NebularNerd** [comment]
I'm getting the same issue, removed all drivers and installed those recommended on the main page. PotatoNV appears to flash on my Kirin 960 but then timeout's on the reboot.

![image](https://github.com/mashed-potatoes/PotatoNV/assets/8470449/dcb3a597-3049-4e95-9457-16759f385167)

**Comment by Crimson1984** [comment]
Same problem, but I solved it.
You might need to install adb & fastboot driver for your device.

**Reply by NebularNerd** [reply]
Mine was super dead, tried many solutions on both Windows and Linux, I had all the fastboot stuff installed etc...

I had three outcomes depending on the firmware I was trying:

1.  It would reboot then timeout, Windows would show a device descriptor failure. You had to unplug the table then reconnect to try again.
2. It would reboot back to COM1 mode, still timeouts obviously.
3, It would freeze or error when presented with certain firmware's.

I was trying some other stuff over on [XDA (I am Oberth on there, same profile pic as here](https://xdaforums.com/t/rom-13-0-lineageos-20-huawei-mediapad-m5-sht-cmr-update-15-06-2025.4682138/page-10), so I reached the conclusion that my tablet had just decided it had enough of life. It's not all doom and gloom, I now have a Xiaomi Pad 7 Pro and that's an amazing bit of kit.
